openedx/core/djangoapps/schedules/management/commands/send_course_due_date_reminders.py

```diff
@@ -0,0 +1,40 @@
+"""
+Management command to send Schedule course due date reminders
+"""
+
+import datetime
+import pytz
+from textwrap import dedent  # lint-amnesty, pylint: disable=wrong-import-order
+
+from django.contrib.sites.models import Site
+
+from openedx.core.djangoapps.schedules.management.commands import SendEmailBaseCommand
+from openedx.core.djangoapps.schedules.tasks import COURSE_DUE_DATE_REMINDER_LOG_PREFIX, ScheduleCourseDueDateReminders
+
+
+class Command(SendEmailBaseCommand):
+    """
+    Command to send Schedule course updates for Self-paced Courses
+    """
+    help = dedent(__doc__).strip()
+    async_send_task = ScheduleCourseDueDateReminders
+    log_prefix = COURSE_DUE_DATE_REMINDER_LOG_PREFIX
+
+    def add_arguments(self, parser):
+        super().add_arguments(parser)
+        parser.add_argument(
+            '--due-in',
+            type=int,
+            help='Remind subsections due in given days',
+        )
+
+    def handle(self, *args, ** options):
+        current_date = datetime.datetime(
+            *[int(x) for x in options['date'].split('-')],
+            tzinfo=pytz.UTC
+        )
+
+        site = Site.objects.get(domain__iexact=options['site_domain_name'])
+        override_recipient_email = options.get('override_recipient_email')
```

Comment:
Could you please add these optional arguments to the docstring?

It would also be nice to add an example usage there (like, e.g., [this one](https://github.com/openedx/edx-platform/blob/ea26268b0a07f44add8a7d9d11c25d4b82bcdfa4/lms/djangoapps/program_enrollments/management/commands/link_program_enrollments.py#L26)).